File: templates/python/motor_deep_response_surface.py
"""motor_deep_response_surface: 电机深化响应等值面（contour 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='electric motor analysis: response contour surface'):
    return make_template_figure('contour', seed=2304, title=title, domain='electric motor analysis', topic='response contour surface')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/motor_deep_response_surface.m
function fig = motor_deep_response_surface()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('contour', 2304, 'electric motor analysis: response contour surface', 'electric motor analysis', 'response contour surface');
end
